README.md

```diff
@@ -89,6 +89,12 @@ fundle plugin 'danhper/fish-fastdir'
 
 will install the repository at https://github.com/danhper/fish-fastdir.
 
+To pick a specific version of the plugins, you can append @ followed by a tag from the repo:
+```
+fundle plugin 'joseluisq/gitnow@2.7.0'
+```
+will install Gitnow release 2.7.0 at https://github.com/joseluisq/gitnow/releases/tag/2.7.0.
```

Comment:
I know this is not the repo you chose as your example though the whole file, but it's the first that came to mind that had versioning and thus fitting this example.

Reply:
Or ilancosman/tide@v4.1.1 or jorgebucaran/nvm.fish@2.1.0.